### PR TITLE
css-four-column-layout - Jacob Husband

### PR DESCRIPTION
**Pull request by jacobhusband**
Screen Shot 2022-08-29 at 4 04 47 PM

### CODE REVIEW

File: css-four-column-layout/layout.css [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+}
+
+.row {
+  display: flex;
+}
+
+.col {
+  width: 100%;
+  display: flex;
+  flex-direction: column;
```
Comment (codingcodymiller) [inline css-four-column-layout/layout.css:26]: You should apply these to a separate class that you can add to the element in addition to the `col` class.

File: css-four-column-layout/layout.css [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  max-width: 910px;
+  margin: auto;
+}
+
+.row {
+  display: flex;
+}
+
+.col {
+  width: 100%;
```
Comment (codingcodymiller) [inline css-four-column-layout/layout.css:24]: There are four of these `col` elements sitting side-by-side, this measurement isn't accurate. Also, you should name your column classes based on their size, so it is explicit what they do when read in the HTML.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Comment by jacobhusband** [comment]
Screen Shot 2022-08-29 at 4 20 53 PM

File: css-four-column-layout/layout.css [hunk 3]
```diff
@@ -20,8 +20,11 @@ img {
...
 }
 
-.col {
-  width: 100%;
+.flex-column {
   display: flex;
   flex-direction: column;
 }
+
+.one-fourth {
```
Comment (codingcodymiller) [inline css-four-column-layout/layout.css:28]: You should make it clear that this is a column class. A good name would be `column-fourth`

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**